### PR TITLE
Top app bar

### PR DESCRIPTION
**Pull request by GuptaEesh**
Top-App-Bar contains:

> Standard

> Fixed

> Dense

> Prominent

> Short

> Condensed Short

topAppBar

### CODE REVIEW

**Review by junaidshaikh-js — COMMENTED** [review]
Good Work!

File: components/top-app-bar.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    background-color: var(--secondary-100);
+    display: flex;
+    justify-content: space-between;
+    padding: 0.3rem;
+    width:45rem;
+}
+.hamburger{
+    background-color: transparent;
+}
+.material-icons:hover{
```
[the remark this answers is not in the document]
Comment (GuptaEesh) [reply components/top-app-bar.css:14]: Yeah! You are right but I added that hover state for:
1. Increasing readability.
2. If I need to add something more to hover I needn't add hover state again.

File: components/top-app-bar.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
+
+
+.top-app-bar {
+    align-items: center;
+    background-color: var(--secondary-100);
+    display: flex;
+    justify-content: space-between;
+    padding: 0.3rem;
+    width:45rem;
```
Comment (Hamdrive) [inline components/top-app-bar.css:9]: give spacing

File: components/top-app-bar.css [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+.hamburger{
+    background-color: transparent;
+}
+.material-icons:hover{
+    cursor: pointer;
+}
+.title-flex {
+    
+    flex: 1;
+}
```
Comment (Hamdrive) [inline components/top-app-bar.css:20]: Check formatting of code

File: components/top-app-bar.css [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+.name-for-topAppBar {
+    flex: 5;
+    justify-self: start;
+}
+
+.right-topapp-icons {
+    display: flex;
+    flex: 1;
+    justify-content: space-between;
+    margin: 0.125rem 1rem 0 0;
```
Comment (Hamdrive) [inline components/top-app-bar.css:31]: For small values you can instead use `px`

**Review by Hamdrive — COMMENTED** [review]
Looks good, suggestions given

File: components/top-app-bar.css (hunk 2 above)
Comment (GuptaEesh) [reply components/top-app-bar.css]: done✅

File: components/top-app-bar.css (hunk 3 above)
Comment (GuptaEesh) [reply components/top-app-bar.css]: done✅

File: components/top-app-bar.css (hunk 4 above)
Comment (GuptaEesh) [reply components/top-app-bar.css]: done✅

**Pull request merged**